### PR TITLE
I add an example of how to use the device plugin to demonstrate how g…

### PR DESCRIPTION
**Pull request by flachica**
1º I add an example of how to use the device plugin to demonstrate how global variables are handled in the initialization of the project. +INFO: http://forum.quasar-framework.org/topic/2077/documentation-review

2º I indicate the path of the vue-i18n plugin with which the proprietary components are internationalized

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [X] Feature: Add one example on documentation page
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [X] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [X] It's submitted to the `dev` branch and _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested with all major browsers

If adding a **new feature**, the PR's description includes:
- [X] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

- I add an example of how to use the device plugin to demonstrate how global variables are handled in the initialization of the project. +INFO: http://forum.quasar-framework.org/topic/2077/documentation-review

- I try to be more explicit on the internacionalization documentation process

### CODE REVIEW

File: source/guide/cordova-plugins.md [hunk 1]
```diff
@@ -147,3 +147,61 @@ export default {
...
+<script>
+export default {
+  data () {
+    return {
+      IMEI: ''
+    }
+  },
+  created () {
+    if (typeof device === 'undefined') {
+      this.IMEI = 'Run this on one mobile'
```
Comment (nothingismagick) [inline source/guide/cordova-plugins.md]: Run this on a mobile device.

File: source/guide/cordova-plugins.md [hunk 2]
```diff
@@ -147,3 +147,61 @@ export default {
...
+$ cordova plugin add cordova-plugin-device
+```
+
+Now let's put this plugin to some good use. If you need the information of your device when starting the application, you will have to capture the created event. In one of your Quasar project's pages/layouts/components Vue file, we write:
+
+```html
+// some Vue file
+// remember this is simply an example;
+// only look at how we use the API described in the plugin's page;
+// the rest of things here are of no importance
```
Comment (nothingismagick) [inline source/guide/cordova-plugins.md]: I would either leave out this commentary or put some real vue-esque cruft in here.

File: source/guide/cordova-plugins.md [hunk 3]
```diff
@@ -147,3 +147,61 @@ export default {
...
+```
+globals: {
+    'ga': true, // Google Analytics
+    'cordova': true,
+    'device': true,
+    '__statics': true
+  }
+```
+
+We read the instructions on how to install this plugin. It's always a Cordova command. **So we "cd" into `/src-cordova`** (which is a Cordova generated folder) **and issue the install command form there**:
```
Comment (nothingismagick) [inline source/guide/cordova-plugins.md]: Read the instructions...
issue the install command **from** there

File: source/guide/cordova-plugins.md [hunk 4]
```diff
@@ -147,3 +147,61 @@ export default {
 }
 </script>
 ```
+
+### Example: Device
+First step is to read the documentation of the Cordova plugin that we want to use. We look at [Cordova Plugins list](https://cordova.apache.org/docs/en/latest/#plugin-apis) and click on [Device doc page](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-device/index.html).
```
Comment (nothingismagick) [inline source/guide/cordova-plugins.md]: Look at the [Cordova...

File: source/guide/cordova-plugins.md [hunk 5]
```diff
@@ -147,3 +147,61 @@ export default {
...
+
+This plugin initializes a global variable called `device` which describes the device's hardware and software. As Quasar is the one who captures this event, you need modify the .eslintrc.js file, leaving the property globals in this way:
+
```
Comment (nothingismagick) [inline source/guide/cordova-plugins.md]: As Quasar is captures this event, you need to modify the .eslintrc.js file, with the following global values:

File: source/guide/cordova-plugins.md [hunk 6]
```diff
@@ -147,3 +147,61 @@ export default {
...
+```
+globals: {
+    'ga': true, // Google Analytics
```
Comment (nothingismagick) [inline source/guide/cordova-plugins.md]: I am not sure we should put the **ga** here... Do we reference this anywhere else, i.e. where to put the ga id?

**Comment by rstoenescu** [comment]
Awesome. Thanks for the additions!

**Pull request merged**